### DISCUSSION TITLE
Memory Leak probably in combination with emotion

### DISCUSSION THREAD

**Discussion by arturhenryy** [vercel/next.js#14924] [Help]
Hi, so we are having a memory leak, it was probably there since we started developing the app so it is really hard to pinpoint where this error is coming from, maybe someone has experienced something similar. I took two snapshots and between i run:

Running 1000 requests test @ http://localhost:3000/
50 connections

the allocation of memory is quite significant with around 300mb as you can see here

<img width="1476" alt="Screenshot 2020-07-07 at 16 35 36" src="https://user-images.githubusercontent.com/45144968/86797587-1fc38980-c070-11ea-9bf7-47120bdbd97f.png">

if i select Objects allocated between Snapshot 3 and Snapshot 4

you see the larget position are strings which look like this

<img width="1469" alt="Screenshot 2020-07-07 at 16 39 02" src="https://user-images.githubusercontent.com/45144968/86797793-63b68e80-c070-11ea-9312-a5f138e52825.png">

i suspect this is having to do something with emotion. Why are to strings not garbage collected?

**Comment by IRediTOTO** [comment]
Are you running npm run dev ?
